package.json

```diff
@@ -14,7 +14,7 @@
     "format": "prettier --write ./",
     "build:css": "postcss src/style/tailwind.css -o src/style/main.css",
     "watch:css": "postcss src/style/tailwind.css -o src/style/main.css --watch",
-    "deploy": "aws s3 sync ./dist/ s3://answerking-bucket-react --exclude '*.js' --delete && aws s3 sync ./dist/ s3://answerking-bucket-react --exclude '*' --include '*.js' --content-type application/javascript"

```

Comment:
Turns out my Registry Editor on my laptop has JS files set to content-type 'text-plain', so AWS was automatically setting all Javascript uploaded to S3 as type 'text-plain'. However, for the JavaScript to work it must be content-type 'application/JavaScript'. My initial solution was to firstly deploy all files except the JavaScript files, and then secondly deploy the JavaScript file with the correct type - which is quite long-winded, but the only solution that AWS CLI allows. Anyway, turns out all I had to do was change the type value in the Registry Editor to 'application/JavaScript' and now it works fine.